src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -92,7 +109,33 @@ class VeteranBenefitSummaryLetter extends React.Component {
           recent periods of service.
         </p>
         <h2>Military Service Information</h2>
-        {militaryServiceInfo}
+        <div className="form-checkbox">
+          <input
+              autoComplete="false"
+              id="serviceInfoCheckboxId"
```

Comment:
This is a little brittle--I  believe the request body for posting the BSL options expects different keys from the ones returned in benefitInfo and serviceInfo, so I"ll add a map in helpers.js to define those and look them up in the change handler.